File: services/hospitalManagement/src/main/java/com/poc/medhead/util/response/HospitalResponse.java
package com.poc.medhead.util.response;

import com.poc.medhead.model.MedicalSpeciality;
import lombok.*;
import java.util.Set;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class HospitalResponse {

    private Integer id;
    private String nomOrganisation;
    private String adresse;
    private String codePostal;
    private Set<MedicalSpeciality> specialitesMedicales;
    private Integer litsDisponible;
    private float longitude;
    private float latitude;
    private double distance;
}

File: poc-medhead-frontend/src/app/services/hospital_service/models/hospital-response.ts
/* eslint-disable */
import { MedicalSpeciality } from '../models/medical-speciality';
export interface HospitalResponse {
  adresse?: string;
  codePostal?: string;
  distance?: number;
  id?: number;
  latitude?: number;
  litsDisponible?: number;
  longitude?: number;
  nomOrganisation?: string;
  specialitesMedicales?: Array<MedicalSpeciality>;
}
